apps/tsilhqotin-language-hub/src/Components/Pages/Apps/AppsDetail.tsx

```diff
@@ -0,0 +1,101 @@
+import { Card, CardActions, CardContent, CardMedia, Typography } from '@mui/material';
+
+export function AppsDetail() {
+    const appData = [
```

Comment:
You need to type this Array. 

```ts
type AppInfo = {
name: string;
...
}

const appData: AppInfo[] = [...];
```

Note that you'll probably have a nested data type:
```ts
type AppLinks = {
google: string;
ios: string;
web: string;
}
```